src/requests.rs

```diff
@@ -0,0 +1,117 @@
+use crate::auth::{get_api_app_context, get_api_user_context};
+use std::collections::HashSet;
+use time::{Date, OffsetDateTime};
+use twitter_v2::id::NumericId;
+use twitter_v2::query::Exclude::Replies;
+use twitter_v2::query::TweetField::{AuthorId, CreatedAt, InReplyToUserId};
+
+const MINIMUM_NUMBER_OF_RESULTS: usize = 5;
+const MAXIMUM_NUMBER_OF_RESULTS: usize = 100;
+const DEFAULT_BADGE_SIZE: usize = 10;
+
+// Gets id of currently authorized user.
+#[allow(dead_code)]
```

Comment:
po co ten allow?

Reply:
Póki co we wstępnej wersji kodu tego typu allowy zostawiałem przy każdej funkcji. W finalnej (dzisiejszej) nie będzie ich nigdzie.